CoreInteractiveGraph/nbproject/project.xml

```diff
@@ -102,6 +102,14 @@
                         <specification-version>1.0.6</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
```

Comment:
Has this file changed or just moved the lines around? If just moved then best to revert

Reply:
You're right no change, but autogenerated so probably best to leave it.